<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PicturesIcanView</class>
 <widget class="QWidget" name="PicturesIcanView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>781</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
background:rgb(81,119,128);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>16</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_7
{
color:white;
font-size:24px;
}</string>
    </property>
    <property name="text">
     <string>Pictures I can View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Signup">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:rgb(214,182,151);
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Signout</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>460</y>
     <width>781</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
background:rgb(81,119,128);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>11</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton
{
background:rgb(81,119,128);
border-radius:60px;
color:black;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Pictures I can view</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton
{
background:rgb(81,119,128);
border-radius:60px;
color:black;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Requests</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton
{
background:rgb(81,119,128);
border-radius:60px;
color:black;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>My Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton
{
background:rgb(81,119,128);
border-radius:60px;
color:black;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Home Page</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton
{
background:rgb(81,119,128);
border-radius:60px;
color:black;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Profile I can view</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>59</y>
     <width>771</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
background:#333;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>0</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>200</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:green;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:green;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:red;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Request Again</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:red;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Request Again</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:red;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Request Again</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:green;
border-radius:60px;
color:white;
}
QPushButton:hover
{
background:argb(214,182,151);
border-radius:60px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
